bench/commands/utils.py

```diff
@@ -116,8 +116,9 @@ def backup_all_sites():
 @click.option('--develop', default='develop')
 @click.option('--master', default='master')
 @click.option('--owner', default='frappe')
-def release(app, bump_type, develop, master, owner):
+@click.option('--repo-name')
+def release(app, bump_type, develop, master, owner, repo_name):
 	"Release app (internal to the Frappe team)"
 	from bench.release import release
-	repo = os.path.join('apps', app)
+	repo = os.path.join('apps', repo_name) if repo_name else os.path.join('apps', app)
```

Comment:
@anandpdoshi is this right?
I tested 2 commands on https://github.com/revant/release_test app : 

1. ``bench release --owner revant --repo-name release_test release_test patch``
1. ``bench release --owner revant release_test patch``